From dd7bce03422b8e00bad8895bf47fa85f2a5edb79 Mon Sep 17 00:00:00 2001
From: Nikita Yushchenko <nikita.yoush@cogentembedded.com>
Date: Thu, 25 May 2023 10:46:48 +0600
Subject: [PATCH 68/92] arm64: configs: vc4: update rswitch support

- enable rswitch2 driver instead of rswitch driver
- enable 1G and 2.5G phy drivers

Signed-off-by: Nikita Yushchenko <nikita.yoush@cogentembedded.com>
---
 arch/arm64/configs/renesas-vc4_defconfig | 6 +++++-
 1 file changed, 5 insertions(+), 1 deletion(-)

diff --git a/arch/arm64/configs/renesas-vc4_defconfig b/arch/arm64/configs/renesas-vc4_defconfig
index 8e624e4542ee..14012bb3f3ae 100644
--- a/arch/arm64/configs/renesas-vc4_defconfig
+++ b/arch/arm64/configs/renesas-vc4_defconfig
@@ -283,7 +283,9 @@ CONFIG_LAN865X=y
 # CONFIG_NET_VENDOR_RDC is not set
 # CONFIG_NET_VENDOR_REALTEK is not set
 CONFIG_RAVB=y
-CONFIG_RENESAS_ETHER_SWITCH=y
+CONFIG_RENESAS_RSWITCH2=y
+CONFIG_RENESAS_RSWITCH2_PLATF=y
+CONFIG_RENESAS_RSWITCH2_GWCA=y
 # CONFIG_NET_VENDOR_ROCKER is not set
 # CONFIG_NET_VENDOR_SAMSUNG is not set
 # CONFIG_NET_VENDOR_SEEQ is not set
@@ -300,6 +302,8 @@ CONFIG_RENESAS_ETHER_SWITCH=y
 # CONFIG_NET_VENDOR_VIA is not set
 # CONFIG_NET_VENDOR_WIZNET is not set
 # CONFIG_NET_VENDOR_XILINX is not set
+CONFIG_MARVELL_88Q211X_PHY=y
+CONFIG_MARVELL_88Q3344_ES_PHY=y
 # CONFIG_WLAN is not set
 CONFIG_NET_FAILOVER=y
 # CONFIG_INPUT_LEDS is not set
-- 
2.30.2

